packages/strapi-design-system/src/MainNav/NavFooter.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Box } from '../Box';
+
+const BoxRelative = styled(Box)`
+  position: relative;
+`;
```

Comment:
What do you think about adding a `position` prop to `Box`? I feel like this would help us in many places.

Reply:
I would love that - would do it in another PR though, I'll create a ticket to not forget!